package.json

```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@mdx-js/loader": "^0.15.4",
     "@mdx-js/mdx": "^0.15.4",
+    "@nkbt/react-collapse": "^5.0.0-alpha.3",
```

Comment:
5.0.0 er released, så da bør vi nok heller gå for den versjoen

Reply:
Done